util/testutil/integration/run_windows.go

```diff
@@ -0,0 +1,17 @@
+package integration
+
+var BusyboxImage = "registry.k8s.io/e2e-test-images/busybox:1.29-2"
+
+func officialImages(_ ...string) map[string]string {
+	// basic mapping for now since there are
+	// very few official Windows-based images
+
+	// TODO: still don't understand why I can't use busybox:latest
```

Comment:
SOS: will appreciate any ideas on how I should handle this. When are the mirrored images used?
Locally, I could mirror the images, but then that was the end...
```
 run.go:274: copied registry.k8s.io/e2e-test-images/busybox:1.29-2 to local mirror localhost:55938/busybox:latest
```

Reply:
Images that are used by tests should be mirrored so that every test does not pull them again from the remote registry. Looking at your current tests, it is `mcr.microsoft.com/windows/nanoserver:ltsc2022` that should be mirrored. As this is not an image from Docker Hub (maybe there is alternative name for same image) then it should need updates in `mirrorConfig` as well to support that.

Reply:
found it, needed to supply `library/` prefix for the key.